app/models/form/question.rb

```diff
@@ -75,4 +97,8 @@ def evaluate_condition(condition, case_log)
       raise "Not implemented yet"
     end
   end
+
+  def filter_inferred_answers(inferred_answers, case_log)
```

Comment:
Think it might be better to name this after what it filters by since it isn't a generic filter method? Something that tells you what is actually returned along the lines of:

`enabled_inferred_answers` ?